v2/openapi/fb-provider-transfer-api.yaml

```diff
@@ -1283,8 +1384,274 @@ components:
         - succeeded
         - failed
 
-
     DepositAddressCreationPolicy:
       enum:
         - CanCreate
         - CannotCreate
+
+    ParticipantRelationshipType:
+      description: >-
+        Defines the relationship between the Originator and the Beneficiary in a transaction.
+
+        - FirstParty: The Originator and Beneficiary are the same entity.
+        - SecondParty: The Originator and Beneficiary have a direct relationship.
+        - ThirdParty: The transaction involves an intermediary between the Originator and Beneficiary.
```

Comment:
"Originator" and "beneficiary" should not be capitalized here. And the description for the second and the third are still wrong. I'd remove the descriptions if we cannot come up with anything correct. It's better than having wrong descriptions.